### ISSUE TITLE
[SF] Icon: click event cannot be triggered by "Enter"/"Space" key with screen reader on

### ISSUE THREAD

**Issue by summer2200** [SAP/ui5-webcomponents#11748]
### Describe the bug

This issue happens in Windows OS.

After turning on screen reader, tab to focus on the Icon, press enter or space key, the click event cannot be triggered.

This issue was found by our customer, please take it as a high priority. Thanks!

### Isolated Example

https://stackblitz.com/edit/github-ysh8324t?file=src%2FApp.tsx,src%2Fmain.tsx

### Reproduction steps

1. Open the isolated example in a Window OS browser
2. Tab to focus on the icon, press enter or space key, popover can be opened, pressing esc key, popover can be closed.
3. Turn on screen reader.
4. Do the same in step 2.

### Expected Behaviour

onClick event should be triggered when screen reader on in Windows OS.

### Screenshots or Videos

https://github.com/user-attachments/assets/810e515d-3715-4fd9-8428-a7dbb84423fe

### UI5 Web Components for React Version

2.10.0

### UI5 Web Components Version

2.10.0

### Browser

Chrome

### Operating System

Windows 10 Enterprise

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.

**Comment by dimovpetar** [comment]
Hello @SAP/ui5-webcomponents-topic-p ,

The same issue occurs in the playground with both JAWS and NVDA.

https://sap.github.io/ui5-webcomponents/play/#eyJpbmRleC5odG1sIjp7Im5hbWUiOiJpbmRleC5odG1sIiwiY29udGVudCI6IjwhLS0gcGxheWdyb3VuZC1mb2xkIC0tPlxuPCFET0NUWVBFIGh0bWw-XG48aHRtbCBsYW5nPVwiZW5cIj5cblxuXG5cbjxoZWFkPlxuICAgIFxuICAgIDxzdHlsZT5cbiAgICAgICo6bm90KDpkZWZpbmVkKSB7XG4gICAgICAgIGRpc3BsYXk6IG5vbmU7XG4gICAgICB9XG5cbiAgICBodG1sIHtcbiAgICAgIGZvcmNlZC1jb2xvci1hZGp1c3Q6IG5vbmU7XG4gICAgfVxuICAgIDwvc3R5bGU-XG5cbiAgICBcbiAgICA8c3R5bGU-XG4gICAgICAqOm5vdCg6ZGVmaW5lZCkge1xuICAgICAgICBkaXNwbGF5OiBub25lO1xuICAgICAgfVxuXG4gICAgaHRtbCB7XG4gICAgICBmb3JjZWQtY29sb3ItYWRqdXN0OiBub25lO1xuICAgIH1cbiAgICA8L3N0eWxlPlxuXG4gICAgPG1ldGEgY2hhcnNldD1cIlVURi04XCI-XG4gICAgPG1ldGEgbmFtZT1cInZpZXdwb3J0XCIgY29udGVudD1cIndpZHRoPWRldmljZS13aWR0aCwgaW5pdGlhbC1zY2FsZT0xLjBcIj5cbiAgICA8dGl0bGU-U2FtcGxlPC90aXRsZT5cbjwvaGVhZD5cblxuPGJvZHkgc3R5bGU9XCJiYWNrZ3JvdW5kLWNvbG9yOiB2YXIoLS1zYXBCYWNrZ3JvdW5kQ29sb3IpXCI-XG4gICAgPCEtLSBwbGF5Z3JvdW5kLWZvbGQtZW5kIC0tPlxuXG4gICAgPHVpNS1pY29uIG5hbWU9XCJob21lXCIgbW9kZT1cIkludGVyYWN0aXZlXCI-PC91aTUtaWNvbj5cbiAgICA8IS0tIHBsYXlncm91bmQtZm9sZCAtLT5cbiAgICA8c2NyaXB0IHR5cGU9XCJtb2R1bGVcIiBzcmM9XCJtYWluLmpzXCI-PC9zY3JpcHQ-XG48L2JvZHk-XG5cbjwvaHRtbD5cbjwhLS0gcGxheWdyb3VuZC1mb2xkLWVuZCAtLT5cbiJ9LCJtYWluLmpzIjp7Im5hbWUiOiJtYWluLmpzIiwiY29udGVudCI6Ii8qIHBsYXlncm91bmQtaGlkZSAqL1xuaW1wb3J0IFwiLi9wbGF5Z3JvdW5kLXN1cHBvcnQuanNcIjtcbi8qIHBsYXlncm91bmQtaGlkZS1lbmQgKi9cbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy9kaXN0L0ljb24uanNcIjtcblxuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzLWljb25zL2Rpc3QvaG9tZS5qc1wiO1xuXG5kb2N1bWVudC5xdWVyeVNlbGVjdG9yKFwiW3VpNS1pY29uXVwiKS5hZGRFdmVudExpc3RlbmVyKFwiY2xpY2tcIiwgKCkgPT4geyBhbGVydChcImNsaWNrXCIpOyB9ICk7In19

Best regards,
Petar

**Comment by NakataCode** [comment]
Hello @summer2200,

This looks like the same issue as #11454. Please verify that your Virtual PC cursor mode is OFF (INS + Z, JAWSKEY + Z or for NVDA - INS + SPACE). While navigating with Virtual PC cursor mode ON, keydown and keyup events are intercepted by JAWS/NVDA and will not work.

Can you try that and see if it resolves the issue?

Best Regards,
Nikola

**Comment by summer2200** [comment]
Hi @NakataCode ,

Popover can be opened with VPC mode off. Thanks a lot for your help!

Best Regards,
Summer
